policyengine_us/variables/gov/states/de/tax/income/credits/de_eitc.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class de_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware EITC"
+    unit = USD
+    documentation = "Refundable and non-refundable Delaware EITC page 8"
+    definition_period = YEAR
+    reference = "https://revenuefiles.delaware.gov/2022/PIT-RES_TY22_2022-02_Instructions.pdf"
+    defined_for = StateCode.DE
+
+    def formula(tax_unit, period, parameters):
+        refundable_eitc = tax_unit("de_refundable_eitc", period)
+        non_refundable_eitc = tax_unit("de_non_refundable_eitc", period)
+        return max_(refundable_eitc, non_refundable_eitc)
```

Comment:
This will always return the greater % non-refundable EITC. Instead we need to give the filer the option that minimizes liability. This may require branching; for an example, see the branching case in the [itemization choice](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/gov/irs/income/taxable_income/deductions/tax_unit_itemizes.py).

@nikhilwoodruff do you see an alternative to branching to determine whether Delaware filers should take the refundable or (larger) non-refundable EITC?

Reply:
@nikhilwoodruff friendly ping